.github/workflows/ci.yaml

```
@@ -783,6 +783,49 @@ jobs:
           name: pytest_buckets
           path: pytest_buckets.txt
           overwrite: true
+  audit-licenses:
+    name: Audit licenses
+    runs-on: ubuntu-22.04
+    needs:
+      - info
+      - base
+    if: |
+      needs.info.outputs.requirements == 'true'
```

```suggestion
    if: |
      (needs.info.outputs.requirements == 'true' || needs.info.outputs.test_full_suite == 'true')
      && github.event.inputs.pylint-only != 'true'
      && github.event.inputs.mypy-only != 'true'
```
Maybe something like this? I'd even suggest to remove the `needs.info.outputs.requirements == 'true'` condition and just always run it on `test_full_suite == 'true'`.

---

Currently we run this only when something changes in requirements, which is the moment imo to do this. I don't mind adding it in the full suite since its cheap but idk